ONSdigital/dp-api-tests
# Feature/update dataset api tests

### What

Expected error messages in response body is always lower cased for dataset API, acceptance test updated to reflect this.

### How to review

Make sure acceptance tests for dataset API (publishing and web) pass.

For acceptance tests to pass:
1) Make sure neo4j, mongo db and kafka & zookeeper are running
2) Run `dp-auth-api-stub` with `make debug`
3) Checkout `dp-dataset-api` on `feature/audit-get-unique-instance`
  a) Run: `make acceptance-publishing`
  b) Run: `make acceptance-web`
4) Run `dp-api-tests`
  a) Run: `cd publishing/datasetAPI; go test ./...; cd ../..`
  b) Run: `cd web/datasetAPI; go test ./...; cd ../..`

See dataset api pr [here](https://github.com/ONSdigital/dp-dataset-api/pull/156)

### Who can review

Anyone
